src/Surfnet/AzureMfa/Infrastructure/Controller/DefaultController.php

```diff
@@ -148,7 +151,7 @@ public function authenticationAction(Request $request)
         $user = $this->azureMfaService->startAuthentication(new UserId($nameId));
 
         return new RedirectResponse(
-            $this->azureMfaService->createAuthnRequest($user, $this->authenticationHelper->useForceAuthn())
+            $this->azureMfaService->createAuthnRequest($user, $this->authenticationHelper->useForceAuthn($request))
```

Comment:
It seems that the AuthenticationHelper::useForceAuthn method doesn't want the parameter?

Reply:
Good call, removed the unwarranted paramter from controller & tests